evap/staff/tests/test_tools.py

```diff
@@ -83,6 +83,11 @@ def setUpTestData(cls):
         cls.rewardpointredemption_main = baker.make(RewardPointRedemption, user_profile=cls.main_user)
         cls.rewardpointredemption_other = baker.make(RewardPointRedemption, user_profile=cls.other_user)
 
+    def setUp(self):
+        # merge users changes these instances in such a way that refresh_from_db doesn't work anymore.
+        self.main_user = UserProfile.objects.get(username="main_user")
+        self.other_user = UserProfile.objects.get(username="other_user")
+
     def test_merge_handles_all_attributes(self):
         user1 = baker.make(UserProfile)
         user2 = baker.make(UserProfile)
```

Comment:
Hmm, this doesn't work since they are set up to fail when merging. Adding the two lines that should fix this from the test that expects them to merge successfully also didn't work. I will think about restructuring this whole test data set up.